packages/fuels/package.json

```
@@ -85,5 +85,12 @@
   "devDependencies": {
     "@types/lodash.camelcase": "^4.3.7",
     "@types/rimraf": "^3.0.2"
-  }
+  },
+  "keywords": [
+    "ethereum",
```

Good catch.

Does it make sense to at least start with the key names?

```suggestion
    "ethereum",
    "fuel",
    "sway",
    "forc",
```